interfaces/wp-rest-media.ts

```diff
@@ -0,0 +1,17 @@
+export default interface WPRESTMedia {
+  id?: number;
+  media_details?: {
```

Comment:
Here's a good example of where we don't have control over what's returned and we have to use snake case, because it's coming from a core WP API.